R/doeAnalysis.R

```diff
@@ -405,7 +405,8 @@ doeAnalysis <- function(jaspResults, dataset, options, ...) {
   if (!is.null(jaspResults[["plotPareto"]]) || !options[["plotPareto"]]) {
     return()
   }
-  plot <- createJaspPlot(title = gettext("Pareto Chart of Standardized Effects"))
+  standardizedString <- ifelse(options[["codeFactors"]], gettext("Standardized"), gettext("Unstandardized"))
+  plot <- createJaspPlot(title = gettextf("Pareto Chart of %s Effects", standardizedString), width = 600, height = 400)
```

Comment:
I get that you do it like this, but it's actually easier for translators to do:
```r
  plot <- createJaspPlot(title = if (options[["codeFactors"]]) {
    gettext("Pareto Chart of Standardized Effects")
  } else {
    gettext("Pareto Chart of Unstandardized Effects")
  }, width = 600, height = 400)
```
and (lines 428):
```r
    ggplot2::scale_y_continuous(name = if (options[["codeFactors"]]) {
    gettext("Standardized Effect")
  } else {
    gettext("Unstandardized Effect")
  }, breaks = xBreaks, limits = range(xBreaks)) +
```
it's easier to translate `gettext` than `gettextf` because the people don't have to imagine what the `%s` could be. The extra if statement looks a little ugly but doesn't matter too much. You're free to put the title and name in separate variables of course if you don't want to nest the if statements in the function calls.